### DISCUSSION TITLE
Codes Documentation

### DISCUSSION THREAD

**Discussion by QihaMarz** [BSd3v/Dashboard-Helper#5] [General]
Hello, I would like to request perhaps do you have a documentation regarding this project? I want to study on how do you implement the codes and relate all the functions?

Thank you.

**Comment by BSd3v** [comment]
Hello @QihaMarz,

Have you tried out the app? What questions do you have about the app and how it functions? How are you trying to implement this in a project?

**Reply by QihaMarz** [reply]
Yea I did, it works really well! ☺️ I wanna try to implement it into my Flask app for my school project, so I want to understand how does the functions relate with each other. Anyways, thank you for this!

**Reply by BSd3v** [reply]
Here is the forum topic from the plotly community, maybe it will help.

https://community.plotly.com/t/dashboard-helper/70647

**Reply by QihaMarz** [reply]
Thank you so much for this! I understand it now 😊
